packages/oui-navbar/README.md

```diff
@@ -46,9 +46,8 @@
             icon-class="{{asideLink.iconClass}}"
             on-click="asideLink.onClick"
             ng-repeat="asideLink in $ctrl.asideLinks track by $index"
-            ng-class="asideLink.class"
-            ng-switch="asideLink.name">
-            <oui-navbar-notification ng-switch-when="notifications"
+            ng-class="asideLink.class">
+            <oui-navbar-notification ng-if="asideLink.name === 'notifications'"
```

Comment:
Why did you remove the `switch` ?

Reply:
Multiple directive asking for transclusion